app/main.py

```diff
@@ -2,0 +1,32 @@
+class Animal:
+    alive = []
+
+    def __init__(self, name, health=100,):
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def __repr__(self):
+        return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"
+
+
+class Herbivore(Animal):
+    hidden = True
+
+    def hide(self):
+        if self.hidden is True:
+            self.hidden = False
+        elif self.hidden is False:
+            self.hidden = True
```

Comment:
The `hide` method can be simplified by using `self.hidden = not self.hidden` to toggle the hidden state.